Answers.md

```diff
@@ -1,9 +1,25 @@
 1. If you had to describe semantic HTML to the next cohort of students, what would you say?
 
+semantic HTML is a way to properly organize your html elements rather then use divs everywhere and it's also helpful for seo
+
 2. Describe some differences between ```display: block;``` and ```display: inline;```.
 
+display block creates a block effect around your html which is editable via styling to control spacing, padding and margin
+
+display inline is inside of the block, basically the html element without the ability of adding space or margin around it
+
```

Comment:
you can add spacing and margins on inline elements, display block takes up the entire width of the parent element, inline allows other elements to be beside it